test/e2e/tests/transaction/send-eth.spec.js

```diff
@@ -157,14 +155,6 @@ describe('Send ETH', function () {
       await withFixtures(
         {
           fixtures: new FixtureBuilder().build(),
-          localNodeOptions: [
-            {
-              type: 'anvil',
-              options: {
-                hardfork: 'london',
-              },
-            },
-          ],
```

Comment:
we can remove all the cutstom options as these was done initially when anvil was not run by default in all specs. So now, we have default anvil and default prague